solutionbox/inception/datalab_solutions/inception/_util.py

```diff
@@ -35,6 +35,17 @@ def is_in_IPython():
   except ImportError:
     return False
 
+def default_project():
```

Comment:
surprised inception has to code this function. Almost every package would want this. 

Can this be made into a core datalab function?